src/OrderFormAcceptanceTests.Tests/Features/CreateAnOrder.feature

```diff
@@ -0,0 +1,21 @@
+﻿Feature: Create An Order
+	As a Buyer
+	I want to create an Order
+	So that I can order from the Buying Catalogue
+
+Background: 
+	Given that a User is not authenticated and the user logs in using the 'login' function
+	When the User is a Buyer User
+	Then the User will be logged in
+
+Scenario: Create a new Order
+	Given the User is presented with the Organisation's Orders dashboard
+	And they choose to create a new Order
```

Comment:
When

Reply:
Ah, yes. See next commit, will add comment to story to ask for an update